module1/homework/make_unique/make_unique-ut.cpp

```diff
@@ -17,19 +18,18 @@ struct make_uniqueTest {
     make_uniqueTest(int& intValue, double&& doubleValue, bool boolValue)
     : intValue_(intValue), doubleValue_(doubleValue), boolValue_(boolValue)
     {}
-    make_uniqueTest(int&& intValue, double& doubleValue, bool boolValue, char charValue)
+    make_uniqueTest(int&& intValue, double& doubleValue, bool boolValue)
         : intValue_(intValue), doubleValue_(doubleValue), boolValue_(boolValue)
```

Comment:
```suggestion
        : intValue_(testIntValueR), doubleValue_(testDoubleValueL), boolValue_(boolValue)
```
this is the trick that I wanted to see. Now you check if intValue_ and doubleValue_ are set properly in tests, depending on the way how parameters were passed.

Reply:
ok, finally got it :)
:panda_face: :three: